lib/components/user/places/place-editor.tsx

```diff
@@ -64,15 +65,10 @@ class PlaceEditor extends Component<
     values: any
   } & WrappedComponentProps
 > {
-  _handleLocationChange = ({
-    location
-  }: {
-    location: {
-      lat: number
-      lon: number
-      name: string
-    }
-  }) => {
+  _handleLocationChange = (
+    _: IntlShape, // Ignore intl object. TODO: localize name?
```

Comment:
What do you mean by "localize name"?

Reply:
I've clarified the comment in 36909e9b441bf6b9dc7bea471474ca1d80b39ee2. I'm not sure if this makes sense to do, but thought I could clarify what the `intl` object could possibly be used for one day